Gems/LyShine/Code/Include/LyShine/IDraw2d.h

```diff
@@ -482,6 +482,50 @@ class Draw2dHelper
         return nullptr;
     }
 
+    //! Helper to load a texture
+    static AZ::Data::Instance<AZ::RPI::Image> LoadTexture(const AZStd::string& pathName)
+    {
+        if (gEnv && gEnv->pLyShine) // [LYSHINE_ATOM_TODO][GHI #3569] Remove LyShine global interface pointer from legacy global environment
+        {
+            return gEnv->pLyShine->LoadTexture(pathName);
+        }
```

Comment:
Seems like this could be made into an `AZ::Interface<ILyShine>` that is registered in the LyShineSystemComponent  Activate/Deactivate().

The LyShineSystemComponent could then just store an `AZStd::unique_ptr<ILyShine>` on it.

Reply:
I should have mentioned that this is just temporary. I actually have an incoming PR that does what you've described, but am trying to get this PR merged in before posting that one. (I will update this line before posting the latter PR.)